rpm/assets/modules.yaml

```diff
@@ -82,6 +93,13 @@ data:
   license:
     module:
       - MIT
+  dependencies:
+  - buildrequires:
+      walrus: []
+      platform: [-f29, -f30]
+    requires:
+      walrus: []
+      platform: [-f29, -f30]
```

Comment:
This module requires any platform except F29 or F30 and it requires `walrus` module of any stream.